runtime/verbose/module.xml

```diff
@@ -81,19 +81,19 @@
 			<include path="j9gcinclude"/>
 			<include path="$(OMR_DIR)/gc/include" type="relativepath"/>
 			<include path="j9gcgluejava"/>
-			<include path="../tr.source/runtime" type="relativepath">
+			<include path="../compiler/runtime" type="relativepath">
 				<include-if condition="spec.flags.J9VM_INTERP_NATIVE_SUPPORT"/>
 			</include>
-			<include path="../tr.source/ibm/runtime" type="relativepath">
+			<include path="../compiler/ibm/runtime" type="relativepath">
 				<include-if condition="spec.flags.J9VM_INTERP_NATIVE_SUPPORT"/>
 			</include>
 			<include path="j9codert_vm">
 				<include-if condition="spec.flags.J9VM_INTERP_NATIVE_SUPPORT"/>
 			</include>
-			<include path="../tr.source/codegen" type="relativepath">
+			<include path="../compiler/codegen" type="relativepath">
 				<include-if condition="spec.flags.J9VM_INTERP_NATIVE_SUPPORT"/>
 			</include>
-			<include path="../tr.source/ibm/codegen" type="relativepath">
+			<include path="../compiler/ibm/codegen" type="relativepath">
```

Comment:
This isn't a path that exists even in the current source.  I think it can be deleted.

Reply:
Fixed in 626683a